node/node.cpp

```diff
@@ -25,12 +25,14 @@ namespace
 
 
 node::node(std::shared_ptr<bzn::asio::io_context_base> io_context, std::shared_ptr<bzn::beast::websocket_base> websocket, std::shared_ptr<chaos_base> chaos, const std::chrono::milliseconds& ws_idle_timeout,
-    const boost::asio::ip::tcp::endpoint& ep)
+    const boost::asio::ip::tcp::endpoint& ep, std::shared_ptr<bzn::crypto_base> crypto, std::shared_ptr<bzn::options_base> options)
```

Comment:
Conceptually it could change at runtime (key rotations), but my thinking was just that I'd rather not have the signature of the module have to change every time it requires a new option; you shouldn't have to directly handle all the options it wants to build a crypto. In particular, the chaos module already has six different options. It would be a pain to pass them all directly.